common/util/debug.py

```diff
@@ -54,6 +60,14 @@ def make_log_message(_type, **kwargs):
 
 def log_git(command, stdin, stdout, stderr, seconds):
     """ Add git command details to debug log """
+    global enabled
```

Comment:
This is redundant but it is a `global`. (This is probably just a habit because then with the args and the one global I have all moving variables at sight.)